app/views/drawings/index.html.haml

```diff
@@ -1,5 +1,2 @@
 #drawings
   = render 'drawings'
-
-#paginator.text-center
-  = link_to_next_page @drawings, 'MORE', remote: true, id: 'load_more'
```

Comment:
I was under the impression that this wasn't going to be used because the infinite scrolling wasn't working. I couldn't completely misunderstood that. I'll put it back, comment it out and add a `TODO` for one of us to fix (this infinite scrolling I mean) at a later date